src/common/components/select-repository-list/styles.css

```diff
@@ -5,7 +5,20 @@
 }
 
 .footer {
+  padding: 1rem;
   display: flex;
   justify-content: space-between;
-  margin: 60px 0;
+  align-items: baseline;
+}
+
+.repoList {
+  height: 300px;
+  overflow: auto;
```

Comment:
Making repos list scrollable in a fixed height of 300px.

Height of the list should depend on viewport size, but it's to be implemented separately.